ioredis v5 filename case change breaks import of built/command.ts

I have upgraded ioredis to v5 and noticed that my tests work fine on Mac but not on the CI running Linux. 

```
 Cannot find module 'ioredis/built/command' from '../../node_modules/ioredis-mock/lib/index.js'
```

As it turns out, the casing for the module changed in ioredis v5 from **built/command.ts** to **built/Command.ts**. OSX is case-insensitive, so tests were still working locally but not when I ran them trough the linux-based deployment pipeline.

additional note, just saw this pull request which I believe fails for the described reason: [1163](https://github.com/stipsan/ioredis-mock/pull/1163)

Greets
-act

--- a/package.json
+++ b/package.json
@@ -50,7 +50,7 @@
     "standard-as-callback": "^2.1.0"
   },
   "peerDependencies": {
-    "ioredis": "4.x"
+    "ioredis": "4.x || 5.x"
   },
   "devDependencies": {
     "@babel/cli": "^7.16.8",

--- a/src/command.js
+++ b/src/command.js
@@ -1,7 +1,17 @@
-import IoredisCommand from 'ioredis/built/command'
 import asCallback from 'standard-as-callback'
 
 import promiseContainer from './promise-container'
+
+let IoredisCommand
+try {
+  // ioredis v4
+  // eslint-disable-next-line global-require, import/no-unresolved
+  IoredisCommand = require('ioredis/built/command').default
+} catch (err) {
+  // ioredis v5
+  // eslint-disable-next-line global-require, import/no-unresolved
+  IoredisCommand = require('ioredis/built/Command').default
+}
 
 export function isInSubscriberMode(RedisMock) {
   if (RedisMock.channels === undefined) {

